test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/XmlDataContractSerializerInputFormatterTest.cs

```diff
@@ -330,9 +330,8 @@ public void SetMaxDepth_ThrowsWhenMaxDepthIsBelowOne()
             // Arrange
             var expectedException = TestPlatformHelper.IsMono ? typeof(SerializationException) :
                                                                 typeof(XmlException);
```

Comment:
Are you removing `TestPlatformHelper.IsMono` use or not? Either do it all (at least in files you're changing) or leave that for another pass.